src/libs/ReportUtils.ts

```diff
@@ -5736,13 +5754,16 @@ function canEditPolicyDescription(policy: OnyxEntry<Policy>): boolean {
 /**
  * Checks if report action has error when smart scanning
  */
-function hasSmartscanError(reportActions: ReportAction[]) {
+function hasSmartscanError(reportActions: ReportAction[], isLHNPreview: boolean) {
     return reportActions.some((action) => {
         if (!ReportActionsUtils.isSplitBillAction(action) && !ReportActionsUtils.isReportPreviewAction(action)) {
             return false;
         }
         const IOUReportID = ReportActionsUtils.getIOUReportIDFromReportActionPreview(action);
-        const isReportPreviewError = ReportActionsUtils.isReportPreviewAction(action) && hasMissingSmartscanFields(IOUReportID) && !isSettled(IOUReportID);
+        const isReportPreviewError =
+            ReportActionsUtils.isReportPreviewAction(action) &&
+            (isLHNPreview ? shouldShowRBRForMissingSmartscanFields(IOUReportID) : hasMissingSmartscanFields(IOUReportID)) &&
```

Comment:
This is a bit hard to read, let's move this into a separate variable, something like
```js
const hasMissingFields = isLHNPreview ? shouldShowRBRForMissingSmartscanFields(IOUReportID) : hasMissingSmartscanFields(IOUReportID);
const isReportPreviewError = ReportActionsUtils.isReportPreviewAction(action) && hasMissingFields && !isSettled(IOUReportID);
```